### PR TITLE
added vlc-client-instance logic

### PR DESCRIPTION
**Pull request by edenhermelin**
* added vlc-client-instance logic
* updated readme to have the basic vlc commands
* fixed the ./ in the launcher

### CODE REVIEW

File: manager.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const {VLCClientInstance} = require('./vlc-client-instance');
+const {VLCLauncher} = require('./vlc-launcher');
+const {logger} = require('./logger');
+const config = require('./config');
+
+const telnetPassword = config.telnetPassword;
+let lastPortInUsed = config.startPort;
```
Comment (Hermel) [inline manager.js]: lastPortInUse

File: config.json [hunk 2]
```diff
@@ -1,3 +1,5 @@
 {
-  "VLCPath": "C:\\Program Files\\VideoLAN\\VLC\\vlc.exe"
+  "VLCPath": "C:\\Program Files\\VideoLAN\\VLC\\vlc.exe",
+  "startPort": 666,
```
Comment (Hermel) [inline config.json]: initialPort

File: manager.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+let vlcId = 1;
+const vlcProcesses = {};
+const vlcClients = {};
+const openNewVlc = async (file) => {
+    let port = lastPortInUsed++;
```
Comment (Hermel) [inline manager.js]: https://stackoverflow.com/questions/971312/why-avoid-increment-and-decrement-operators-in-javascript

**Review by edenhermelin — COMMENTED** [review]
wrote some comments, check them out (hope it'll be readable from your user)

**Pull request merged**